### PR TITLE
[release-4.14] OCPBUGS-45080: SSA for Services and ClusterRoleBindings

### PR DESCRIPTION
**Pull request by dtfranz**
Manual cherry-pick of #707 due to conflicts

### CODE REVIEW

**Comment by dtfranz** [comment]
Getting some possible flakes - e2e has failed twice now but on different tests. Running them again.

**Comment by jianzhangbjz** [comment]
Test pass, details: https://issues.redhat.com/browse/OCPBUGS-45080
/label qe-approved
/label cherry-pick-approved

File: staging/operator-lifecycle-manager/pkg/controller/install/certresources.go [hunk 1]
```diff
@@ -503,50 +495,25 @@ func (i *StrategyDeploymentInstaller) installCertRequirementsForDeployment(deplo
...
-		if err != nil {
-			log.Warnf("could not create auth delegator clusterrolebinding %s", authDelegatorClusterRoleBinding.GetName())
-			return nil, nil, err
-		}
-	} else {
+	// apply ClusterRoleBinding to system:auth-delegator Role
+	crbLabels := map[string]string{OLMManagedLabelKey: OLMManagedLabelValue}
+	for key, val := range ownerutil.OwnerLabel(i.owner, i.owner.GetObjectKind().GroupVersionKind().Kind) {
+		crbLabels[key] = val
+	}
```
[the remark this answers is not in the document]
Comment (dtfranz) [reply staging/operator-lifecycle-manager/pkg/controller/install/certresources.go]: Thanks @joelanford ! I've removed them.

**Comment by jianzhangbjz** [comment]
Retest it, pass.
/lgtm

**Pull request merged**